### PR TITLE
refactor: Update country-wise tax rates in Ethiopia

### PR DESCRIPTION
**Pull request by mubareksd**
- Updated the tax rates for Ethiopia in the country_wise_tax.json file
- Added new VAT rates for 10% and 2%
- Set the default VAT rate to 15%

### CODE REVIEW

File: erpnext/setup/setup_wizard/data/country_wise_tax.json [hunk 1]
```diff
@@ -428,9 +428,18 @@
 	},
 
 	"Ethiopia": {
-		"Ethiopia Tax": {
-			"account_name": "VAT",
-			"tax_rate": 15.00
+		"Ethiopia VAT 15%": {
+			"account_name": "VAT 15%",
```
Comment (vorasmit) [inline erpnext/setup/setup_wizard/data/country_wise_tax.json:432]: Use of % in account name could cause issue with query builder and should be avoided.

Alternatively, did you try using one account with multiple tax rates using item tax template? That's a better approach unless it's legally required for you to maintain rate wise accounts.

**Comment by mubareksd** [comment]
@vorasmit

Regarding the use of one account with multiple tax rates using the item tax template, we understand the suggestion. However, VAT and TOT are fundamentally different types of taxes with distinct implications and regulations. VAT is applicable to certain merchants at a rate of 15% for both products and services, whereas TOT applies to a different set of merchants with rates of 10% for services and 2% for products.

Maintaining separate accounts for VAT and TOT is necessary to ensure accurate tax reporting and compliance with local tax laws.

File: erpnext/setup/setup_wizard/data/country_wise_tax.json (hunk 1 above)
Comment (mubareksd) [reply erpnext/setup/setup_wizard/data/country_wise_tax.json:432]: Regarding the use of one account with multiple tax rates using the item tax template, we understand the suggestion. However, VAT and TOT are fundamentally different types of taxes with distinct implications and regulations. VAT is applicable to certain merchants at a rate of 15% for both products and services, whereas TOT applies to a different set of merchants with rates of 10% for services and 2% for products.

Maintaining separate accounts for VAT and TOT is necessary to ensure accurate tax reporting and compliance with local tax laws.

**Comment by mubareksd** [comment]
:)

**Pull request closed without merging**